tripy/setup.py

```diff
@@ -49,5 +50,10 @@ def generate_stubs(self):
             print(f"Error generating stubs: {e}")
 
 
+default_excludes = list(FlatLayoutPackageFinder.DEFAULT_EXCLUDE) + ["notebooks"]
+
 if __name__ == "__main__":
-    setup(cmdclass={"build_py": BuildPyCommand})
+    setup(
+        cmdclass={"build_py": BuildPyCommand},
+        packages=find_packages(exclude=default_excludes),  # Use the dynamically updated exclude list
```

Comment:
Can you add a comment here explaining why we didn't use `include` in `pyproject.toml`?

Reply:
done